app/src/main/java/cm/aptoide/pt/editorial/EditorialManager.java

```diff
@@ -89,7 +89,7 @@ private void setupDownloadEvents(Download download, String packageName, long app
     editorialAnalytics.setupDownloadEvents(download, campaignId, abTestGroup,
         AnalyticsManager.Action.CLICK, offerResponseStatus);
     installAnalytics.installStarted(download.getPackageName(), download.getVersionCode(),
-        AnalyticsManager.Action.INSTALL, AppContext.EDITORIAL,
+        AnalyticsManager.Action.INSTALL, AppContext.EditorialFragment,
```

Comment:
Will this change the value in the BI Install event? We cannot change this values or we might loose/dimension data in the BI dashboards.